configurator/src/main/resources/stateSetsTabView.fxml

```diff
@@ -92,10 +89,9 @@
                         <TextField fx:id="stateSetId" onAction="#onStateSetIDAction" GridPane.columnIndex="2" GridPane.rowIndex="3" />
                         <TableView fx:id="stateSetValueRecords" prefHeight="200.0" prefWidth="200.0" GridPane.columnIndex="2" GridPane.rowIndex="4">
                           <columns>
-                            <TableColumn fx:id="minStateValue" prefWidth="104.0" text="Min State Value" />
-                            <TableColumn fx:id="maxStateValue" prefWidth="105.0" text="Max State Value" />
-                              <TableColumn fx:id="languageTags" prefWidth="99.0" text="language Tags" />
-                              <TableColumn fx:id="stateName" prefWidth="87.0" text="State Name" />
+                            <TableColumn fx:id="stateValue" prefWidth="125.0" text="State Value" />
+                              <TableColumn fx:id="languageTags" prefWidth="139.0" text="language Tags" />
```

Comment:
I think Mr. Sandy wanted us to remove language tags. I also recall that he no longer wanted multiple columns, such that there is a only a list of state names? I may have misinterpreted this because it seems to contradict the spec requirements